packages/terra-form-select/translations/en-US.json

```diff
@@ -1,5 +1,6 @@
 {
   "Terra.form.select.add": "Add \"{text}\"",
+  "Terra.form.select.close": "Close",
```

Comment:
A new en-CA translation file just got merged today: https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/translations/en-CA.json
Can you also add this string translation to the en-CA file?

Reply:
This PR might end up being scrapped. Since we've hit a lot of regressions with existing functionality, we're exploring new options going forward with UX.

If we continue down this path, I'll be sure to add it though!